docs/app-design.md

```diff
@@ -76,20 +73,31 @@
 * Emails are sent to users who entered the errata as it moves through the workflow. 
 * Errata can be filtered in the UI
 
+### Eventbrite
+* Wrappers around the eventbrite API
+
+### Events
+* API to register and view Eventbrite event registrations
+
 ### Extra Admin Filters
 * Used to filter errata
 * Allows users to filter multiple books in the same query
 
+### Fixtures
```

Comment:
You can also use fixtures to mock up test data, just FYI 
https://docs.djangoproject.com/en/4.0/howto/initial-data/